src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import { render } from '../shared/dom';
+
+
+
+const template = (className, onClick, text, type) => `
```

Comment:
What about styling?
Would you like to add it in other task or here?

Reply:
I forgot, I'll add it here tomorrow